packages/asset/contracts-temp/AssetMinter.sol

```diff
@@ -24,10 +24,7 @@ contract AssetMinter is
     using TokenIdUtils for uint256;
     address public assetContract;
     address public catalystContract;
-    bytes32 public constant REVEAL_TYPEHASH =
-        keccak256(
-            "Reveal(address creator,uint256 prevTokenId, uint256 amount,string[] metadataHashes)"
-        );
+
```

Comment:
might be better to pass those as arguments of initialize
```
    string public constant name = "Sandbox Asset Minter";
    string public constant version = "1.0";
```

Reply:
Interesting comment, from my perspective there is no need to pass those during initilization, can you elaborate what is the use for moving that into initialize?

This is a constant and can only have one unchangeable value, if we will be changing the implementation all we going ot change during the code changes is like `1.0` => `1.1` or something.

Reply:
Oh I didn't notice its AssetMinter, please note asset minter contract will go away/be heavily changed soon.

Reply:
What's bothering me is that those constant already exist in EIP712Upgradeable
https://github.com/OpenZeppelin/openzeppelin-contracts-upgradeable/blob/master/contracts/utils/cryptography/EIP712Upgradeable.sol#L44

Reply:
and i can return the question, why would you want to add constant for that when there are already variables that save that

Reply:
I see, but that't not the contract version we use in our repo currently.

You're looking at a version that was released 3 days ago, when we installed the package it was different. 😅
https://github.com/OpenZeppelin/openzeppelin-contracts-upgradeable/commit/de14df6d993af38a68db6fdb29342f9c0116b46e

Reply:
Are you saying we should upgrade to newest version?